cmd/kube-proxy/app/server_linux_test.go

```
@@ -43,6 +43,11 @@ import (
 	"k8s.io/utils/ptr"
 )
 
+func parseCIDR(cidrStr string) *net.IPNet {
+	_, cidr, _ := netutils.ParseCIDRSloppy(cidrStr)
```

ignoring the error does not sound right, we can pass the t.Testing and fail or do mustParseCIDR(c string) and panic inside

---

I prefer the later as is how netip does for tests https://cs.opensource.google/go/go/+/refs/tags/go1.21.5:src/net/netip/netip.go;l=132

---

That is intentional. cidr becomes nil on an error, and that is actually tested in some test-cases

---

Sorry, I confused it with pkg/proxy/util/iptables/traffic_test.go, where it _is_ intentional. Now updated as you suggested.